README.md

```diff
@@ -47,6 +47,29 @@ Then open browser at [`http://localhost:3000/`](http://localhost:3000/).
 ├── tests.karma.js              # (self explanatory)
 └── webpack.config.js           # (self explanatory)
 ```
+## Features
+
+* [React](https://facebook.github.io/react/) (`^0.14.6`)
```

Comment:
Not sure about hard-referencing version numbers in this file... every time they change in `package.json`, this file will have to be updated (and I doubt I'd remember to do it!).